components/tel/Core/Src/main.c

```diff
@@ -23,8 +23,10 @@
 #include "dma.h"
 #include "iwdg.h"
 #include "rtc.h"
+#include "tim.h"
 #include "usart.h"
 #include "gpio.h"
+#include "cpu_load.h"

```

Comment:
This include **needs** to go under `/* Private includes ` below or else your include will get removed upon the next autogeneration.